src/components/Button/_button.scss

```diff
@@ -26,97 +26,93 @@
 .btn--secondary,
 .btn--ghost {
```

Comment:
To avoid confusion between different platforms this class should be renamed as `.btn--tertiary`. Likewise all associated classes, variables and props should be `tertiary` not `ghost`.